Description: Fix manpage issues
 Shipped manpages have issues on Debian:
   + .NL macro not supported. Use .BR.
   + NAME values not formatted correctly for lexgrog
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: no
Last-Update: <2018-06-23>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Esm/ib/man/opafmcmd.manPage
+++ b/Esm/ib/man/opafmcmd.manPage
@@ -8,33 +8,33 @@
 
 .TH opafmcmd 8 "Intel Corporation" "Copyright(C) 2015\-2018" "Master map: IFSFFCLIRG (Man Page)" 
 .SH NAME
-opafmcmd
+opafmcmd \- command line utility
 
-.NL
+.BR
 
 .PP
 Executes a command to a specific instance of the Fabric Manager (FM). This command can be used, for example, to query the configuration attributes of a particular instance of an FM as well as issue subnet management commands.
 .SH Syntax
-.NL
+.BR
 opafmcmd [-i \fIfm\(ulinstance\fR] cmd [ \fIargs\fR]
 .SH Options
-.NL
+.BR
 
 .TP 10
 --help 
-.NL
+.BR
 Produces full help text.
 
 .TP 10
 -i \fIfm\(ulinstance\fR 
-.NL
+.BR
 Specifies the number of the FM instance to act on.
 
 .IP
 Range = 0 to 7. Default = 0.
 .TP 10
 cmd [args] 
-.NL
+.BR
 Specifies the commands and arguments (if applicable) to be run. Values include:
 
 .RS
@@ -458,7 +458,7 @@
 .RE
 
 .SH Example 
-.NL
+.BR
 opafmcmd smForceSweep
 .br
 opafmcmd -i 2 smLogLevel 3
--- a/Esm/ib/man/opafmcmdall.manPage
+++ b/Esm/ib/man/opafmcmdall.manPage
@@ -8,33 +8,33 @@
 
 .TH opafmcmdall 8 "Intel Corporation" "Copyright(C) 2015\-2018" "Master map: IFSFFCLIRG (Man Page)" 
 .SH NAME
-opafmcmdall
+opafmcmdall \- command line utility
 
-.NL
+.BR
 
 .PP
 Executes a command to all instances of the Fabric Manager (FM) listed in the options. The behavior is similar to opafmcmd.
 .SH Syntax
-.NL
+.BR
 opafmcmdall [-i \fIfm\(ulinstance\fR] \fIcmd\fR[ \fIargs\fR]
 .SH Options
-.NL
+.BR
 
 .TP 10
 --help 
-.NL
+.BR
 Produces full help text.
 
 .TP 10
 -i \fIfm\(ulinstance\fR 
-.NL
+.BR
 Specifies the FM instance number (0-7) to act on.
 
 .IP
 Specify multiple -i options as needed. Default = all instances.
 .TP 10
 cmd [args] 
-.NL
+.BR
 Specifies the commands and arguments (if applicable) to be run. Values include:
 
 .RS
@@ -458,7 +458,7 @@
 .RE
 
 .SH Example 
-.NL
+.BR
 opafmcmdall smForceSweep
 .br
 # sends command to all FMs
